docs/guides/component-testing/third-party-definitions.mdx

```diff
@@ -0,0 +1,251 @@
+
+By default, Cypress will look for this as a `mount` function that is a **named export** from the package. This should be written in a `index.mjs` file. This example is for a [Solid.js](https://www.solidjs.com/) mount adapter:
+
+```js
+import { getContainerEl, setupHooks } from "@cypress/mount-utils";
+import { render } from "solid-js/web";
+
+let dispose;
+
+function cleanup() {
+  dispose?.();
+}
+
+/**
+ * @param {() => JSX.Element} - component to render
+ */
+export function mount(component, options = {}) {
+  // Retrieve root DOM element that Cypress has prepared for this test
+  const root = getContainerEl();
+
+  dispose = render(() => component, root);
+
+  // Wait until next microtick to ensure any async render logic has executed
+  return cy.wait(0, { log: false }).then(() => {
+    if (options.log) {
+      Cypress.log({
+        name: "mount",
+        message: "Mounted component",
+      });
+    }
+  });
+}
+
+// Cleanup between each test
+setupHooks(cleanup);
+```
+
+This is different for each library, but the concept is the same - identify how to mount or render a component in your library, and implement it in a function named `mount` in `index.mjs`.
+
+When a user configures Component Testing with your Framework Definition we automatically configure a `cy.mount` command using your `mount` function in the Component Testing `supportFile`:
+
+```js
+import { mount } from '@lmiller1990/cypress-ct-solid-js'
+
+Cypress.Commands.add('mount', mount)
+```
+
+## package.json
+
+The final thing you need is a correctly configured `package.json`. This example was created using [our official template](https://github.com/cypress-io/cypress-ct-definition-template).
+
+There are two fields of note:
+* `name`: Your package name which must follow the `cypress-ct-*`/`@org/cypress-ct-*` convention
+* `exports`: Object referencing the two files we created to comprise the Framework Definition. The `node` entry points to the Definition file, and the `default` entry points to the Mount Adapter:
+
+```json {2,5-8}
+{
+  "name": "@lmiller1990/cypress-ct-solid-js",
+  "version": "0.0.4",
+  "description": "Example Framework Definition for Cypress and Solid.js",
+  "exports": {
+    "node": "./definition.cjs",
+    "default": "./index.mjs"
+  },
+  "files": [
+    "package.json",
+    "definition.cjs",
+    "index.mjs"
+  ],
+  "dependencies": {
+    "@cypress/mount-utils": "^4.0.0"
+  },
+  "devDependencies": {
+    "solid-js": "^1.6.0"
+  },
+  "peerDependencies": {
+    "solid-js": "^1.6.0"
+    "cypress": "^12.7.0"
+  }
+}
+```
+
+## Testing
+
+If you develop the Framework Definition file using TypeScript, for example by using [our official template](https://github.com/cypress-io/cypress-ct-definition-template), as long as you don't have any compile time errors, everything should work as expected. If you do run into unexpected behavior, please [file an issue](https://github.com/cypress-io/cypress/issues).
```

Comment:
Worth noting we've got better warnings for definitions coming VERY soon (next sprint) https://github.com/cypress-io/cypress/issues/25838 but they are not supported **right now**. The issue requires the docs to be updated when they are added.